types/src/rsmt2d.rs

```diff
@@ -601,7 +633,7 @@ mod tests {
                 parity_share.clone(),
                 // row 1
                 share(1), // ODS
-                share(1), // ODS
+                share(1), // ODS - error: smaller namespace in 2nd column
```

Comment:
I would like 1 extra example that passes in this case. For example with:

```rust
                // row 0
                share(1), // ODS
                share(2), // ODS
                parity_share.clone(),
                parity_share.clone(),
                // row 1
                share(1), // ODS
                share(3), // ODS
                parity_share.clone(),
                parity_share.clone(),
```

Reply:
added